setup.py

```diff
@@ -0,0 +1,15 @@
+# -*- coding: utf-8 -*-
+from setuptools import setup
+
+setup(
+    name="linked_list",
```

Comment:
This package should probably actually be called "data_structures".  It's more than just a linked list, no?